test/Microsoft.DotNet.Cli.Utils.Tests/PathUtilityTests.cs

```diff
@@ -21,6 +21,32 @@ public void GetRelativePathWithCaseInsensitiveDrives()
             Assert.Equal(@"d:\foo\", PathUtility.GetRelativePath(@"C:\foo\", @"d:\foo\"));
         }
 
+        [WindowsOnlyFact]
+        public void GetRelativePathForFilePath()
+        {
+            Assert.Equal(
+                @"mytool\1.0.1\mytool\1.0.1\tools\netcoreapp2.1\any\mytool.dll",
+                PathUtility.GetRelativePath(
+                    @"C:\Users\myuser\.dotnet\tools\mytool.exe",
+                    @"C:\Users\myuser\.dotnet\tools\mytool\1.0.1\mytool\1.0.1\tools\netcoreapp2.1\any\mytool.dll"));
+        }
+
+        [WindowsOnlyFact]
+        public void GetRelativePathRequireTrailingSlashForDirectoryPath()
+        {
+            Assert.NotEqual(
+                @"mytool\1.0.1\mytool\1.0.1\tools\netcoreapp2.1\any\mytool.dll",
```

Comment:
What is supposed to happen when the trailing slash is missing?

Reply:
I don't know. It is reasonable. But the point of the test is I want to use this tests to doc that piece of code which existed for a long time